docs/js-ref.md

```diff
@@ -358,85 +358,157 @@ aragon.init(() => {
 })
 ```
 
+---
 **API**
 
-**init**
+- [init()](#init)
+- [initAcl()](#initacl)
+- [getAppProxyValues(proxyAddress)](#getappproxyvalues)
+- [isApp(app)](#isapp)
+- [initApps()](#initapps)
+- [initForwarders()](#initforwarders)
+- [runApp(sandbox, proxyAddress)](#runapp)
+- [getAccounts()](#getaccounts)
+- [getTransactionPath(destination, methodName, params)](#gettransactionpath)
+- [calculateTransactionPath(sender, destination, methodName, params)](#calculatetransactionpath)
+
+---
+### init
+
 Initialise the wrapper.
 
-Returns Promise<void>
+**Parameters**
+
+None.
+
+**Returns**
+
+[`Promise`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)&lt;void>
```

Comment:
Hmm, these `Promise` types should probably just use `<type>` (in the code block) instead of the escaped characters.

Reply:
Just to be sure I understood you correctly. You meant something like this instead:

```
<void>
```

```
<Object>
```